### PR TITLE
Coderepo samples 

### PR DESCRIPTION
**Pull request by RahulMR42**

### CODE REVIEW

File: oci-coderepo-examples/oci-devops-coderepo-with-github/README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+--------
+
+#### Objectives
+
+- Create a sample repo and personal access token via Github.
+- Create vault ,policies as accordingly.
+- Create a devops connection and integrate with OCI.
+- Test and validate the integrations.
+a
```
Comment (ashok-cm) [inline oci-coderepo-examples/oci-devops-coderepo-with-github/README.md:12]:
```suggestion
```

File: oci-coderepo-examples/README.md [hunk 2]
```diff
@@ -2,9 +2,19 @@ All about OCI devops coderepo samples ..
...
+* [Integrate GITLAB.com repo with OCI Devops .](./oci-devops-coderepo-with-gitlab/)
+
+</details>
+
+
+
+### Back to examples.
+----
+
+- 🏝️ [Back to OCI Devops sample](../README.md)
```
Comment (timclegg) [inline oci-coderepo-examples/README.md:17]: This is a duplicate.

**Review by timclegg — CHANGES_REQUESTED** [review]
I provided a few minor suggestions.

File: oci-coderepo-examples/oci-devops-coderepo-with-github/README.md (hunk 1 above)
Comment (RahulMR42) [reply oci-coderepo-examples/oci-devops-coderepo-with-github/README.md:12]: Done ,thank you

File: oci-coderepo-examples/README.md (hunk 2 above)
Comment (RahulMR42) [reply oci-coderepo-examples/README.md:17]: Done ,thank you, too much of sleep of me :)

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+
+# How Has This Been Tested?
+
+
+- [ ] Free Tier
+- [ ] Always Free
+- [ ] Paid 
+- [ ] Oracle internal tenancies/BOAT Users
```
[the remark this answers is not in the document]
Comment (RahulMR42) [reply pull_request_template.md:24]: Done ,thank you.

**Comment by RahulMR42** [comment]
PR comments are fixed , please review - @ashok-cm  @timclegg /thank you

**Comment by ashok-cm** [comment]
@RahulMR42  The changes look good.

**Review by ashok-cm — APPROVED** [review]

**Review by timclegg — APPROVED** [review]
Great work!

**Pull request merged**